bundles/net.tourbook/src/net/tourbook/ui/tourChart/action/ActionTourChartPauses.java

```diff
@@ -15,31 +15,182 @@
  *******************************************************************************/
 package net.tourbook.ui.tourChart.action;
 
+import static org.eclipse.swt.events.SelectionListener.widgetSelectedAdapter;
+
 import net.tourbook.Images;
 import net.tourbook.Messages;
 import net.tourbook.application.TourbookPlugin;
+import net.tourbook.common.UI;
+import net.tourbook.common.tooltip.IOpeningDialog;
+import net.tourbook.ui.tourChart.SlideoutTourChartPauses;
 import net.tourbook.ui.tourChart.TourChart;
 
-import org.eclipse.jface.action.Action;
+import org.eclipse.jface.action.ContributionItem;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.graphics.Image;
+import org.eclipse.swt.graphics.Point;
+import org.eclipse.swt.graphics.Rectangle;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.swt.widgets.ToolBar;
+import org.eclipse.swt.widgets.ToolItem;
+
+public class ActionTourChartPauses extends ContributionItem implements IOpeningDialog {
+
+   private String                  _dialogId = getClass().getCanonicalName();
+
+   private TourChart               _tourChart;
 
-public class ActionTourChartPauses extends Action {
+   private ToolBar                 _toolBar;
+   private ToolItem                _actionToolItem;
 
-   private TourChart _tourChart;
+   private SlideoutTourChartPauses _slideoutTourChartPause;
 
-   public ActionTourChartPauses(final TourChart tourChart) {
+   /*
+    * UI controls
+    */
+   private Control _parent;
 
-      super(null, AS_CHECK_BOX);
+   private Image   _imageEnabled;
+   private Image   _imageDisabled;
 
-      setToolTipText(Messages.Tour_Action_ShowTourPauses_Tooltip);
+   public ActionTourChartPauses(final TourChart tourChart, final Control parent) {
 
       _tourChart = tourChart;
+      _parent = parent;
+
+      _imageEnabled = TourbookPlugin.getImageDescriptor(Images.TourPauses).createImage();
```

Comment:
You should use the themed icon, in the dark theme, the pause symbol is almost not visible, also in the 2D map context menu

Themed icon usage in the tour marker action
`_imageEnabled = TourbookPlugin.getThemedImageDescriptor(Images.TourMarker).createImage();`

In the next Eclipse version (tomorrow), the system tooltips should be displayed correctly in the dark theme :-)
https://www.eclipse.org/eclipse/news/4.21/platform_isv.php#swt-win-dark-themed-tooltips

![image](https://user-images.githubusercontent.com/1283445/133214828-587a8fb2-1aec-4ca9-b62b-ba56885da0a7.png)

![image](https://user-images.githubusercontent.com/1283445/133214846-a35c4420-d6e5-459e-b318-3e52cc7e2518.png)

Reply:
Fixed

![image](https://user-images.githubusercontent.com/25235665/133293296-309fc584-a3b7-4db1-ae37-cddae263687a.png)

Reply:
A tooltip is not displayed when the tour pauses slideout is not displayed, when the action is not selected, see the other slideout actions when they are not selected

Reply:
Fixed
![image](https://user-images.githubusercontent.com/25235665/133475230-93c54c62-b9d6-4232-92e1-46245015e2b5.png)